src/domain/accounts/entities/__tests__/create-account.dto.builder.ts

```diff
@@ -5,8 +5,11 @@ import { faker } from '@faker-js/faker';
 import { getAddress } from 'viem';
 
 export function createAccountDtoBuilder(): IBuilder<CreateAccountDto> {
-  return new Builder<CreateAccountDto>().with(
-    'address',
-    getAddress(faker.finance.ethereumAddress()),
+  return (
+    new Builder<CreateAccountDto>()
+      .with('address', getAddress(faker.finance.ethereumAddress()))
+      // Note: regular expression is simplified because faker has limited support for regex.
+      // https://fakerjs.dev/api/helpers#fromregexp
+      .with('name', faker.helpers.fromRegExp(/[a-zA-Z0-9]{12,20}/i))
```

Comment:
Nit: what do you think about extracting this considering it's also used in the `accountBuilder`? Otherwise we could use one of the `word` faker methods?

Reply:
I've extracted it in [2069930](https://github.com/safe-global/safe-client-gateway/pull/2081/commits/2069930ce18518a38b763ad69b1a7585e6f2016e)